docs/platform/concepts.rst

```diff
@@ -16,7 +16,7 @@ Learn about some of the key concepts for working with Aiven platform:
   Learn about Aiven's access control, encryption, network security, data privacy and operator access.
 
 
-* :doc:`Database forking </docs/platform/concepts/database-forking>`.
+* :doc:`Service forking </docs/platform/concepts/service-forking>`.
 
   Quickly and safely make a copy of your database for testing, dry run upgrades, or anything else, without affecting your existing service.
```

Comment:
```suggestion
  Quickly and safely make a copy of your database or other service for testing, dry run upgrades, or anything else, without affecting your existing service.
```
Perhaps clarify here that it is not just a database (but it's nice, I think, to leave the word database in that text).